Basic-Car-Maintenance/Shared/Settings/Views/AddVehicleView.swift

```diff
@@ -62,15 +62,27 @@ struct AddVehicleView: View {
                 }
 
                 Section {
-                    TextField("Vehicle License Plate Number", text: $licensePlateNumber, prompt: Text("Vehicle License Plate Number"))
+                    TextField(
+                        "Vehicle License Plate Number",
+                        text: $licensePlateNumber,
+                        prompt: Text("Vehicle License Plate Number")
+                    )
```

Comment:
Please instead format this by adding the line break after the first parameter, it would look like this:
```swift
TextField("Vehicle License Plate Number",
          text: $licensePlateNumber,
          prompt: Text("Vehicle License Plate Number"))
```

As opposed to adding the line break before the first parameter